kleva-j/iReporter
# Feature/#162502612/sign in a user using a database

#### What does this PR do?
- Enables the sign in of a user using a database.

#### Description of the task to be completed
- Validate and authenticate user details.
- Send authenticated token to the user.

#### How should this be manually tested?
- Clone the repository
- Check out into the branch Feature/#162502612/sign-in-a-user-using-a-database
- Install all dependencies
- run `npm start` to run the app successfully.

#### What are the relevant pivotal tracker stories?
[162502612](https://www.pivotaltracker.com/story/show/162502612)
